### DISCUSSION TITLE
Unicode support in @astrojs/db

### DISCUSSION THREAD

**Discussion by millette** [withastro/roadmap#878] [Proposal]
### Body

# Summary

Sqlite (and libsql) don't support unicode collation out of the box.

- Alpha
- Écho
- X-Ray

gets sorted as

- Alpha
- X-Ray
- Écho

# Background & Motivation

Not really sure if this fits RFC proposals but it's the only place I found to describe the problem.

# Goals

`order by` should support any language, goes hand in hand with I18N.

Usage should be documented for localhost (dev) and it should be supported in the hosted service, out of the box.

**Comment by florian-lefebvre** [comment]
Astro DB has been deprecated
